aire/src/button/button.ts

```diff
@@ -8,6 +8,9 @@ export class AireButton {
   @bindable
   label     : string;
 
+  @bindable
+  icon      : string;
```

Comment:
This is ok for icon-classes but what if we need an image URL?

Reply:
yeah, that's a bit of a todo to go back to -- I noticed we're missing it in some other places where we _only_ allow for svgs, and I think icon font is more likely.